components/bitcoin-topics/Navigation.tsx

```diff
@@ -184,6 +347,43 @@ export function Navigation({
                     </li>
                 ))}
             </ul>
+            <EmailSubscription />
         </nav>
     )
 }
+
+const EmailSubscription = () => {
+    const [email, setEmail] = useState("")
+
+    const handleSubmit = (e: React.FormEvent) => {
+        e.preventDefault()
+        console.log("Submitted:", email)
```

Comment:
I think the console.log is not necessary, also this component is not connected to the subscribe route as it only clears the input value, please connect it

Reply:
good catch on that, impelemented the logic here https://github.com/bitcoin-dev-project/bitcoin-dev-project/pull/102/commits/7e434bd8855108b2f2ed031c64a2d6d231374c27